api/src/main/java/com/google/sps/Application.java

```diff
@@ -5,17 +5,54 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.Firestore;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+
+import java.io.IOException;
+
 @SpringBootApplication
 @RestController
+@WebServlet("/notify")
 public class Application {
 
 	public static void main(String[] args) {
```

Comment:
indentation should be consistent for better readability

Reply:
i dont think i see what you mean, im looking through and it looks like all my indentation is consistent

Reply:
maybe it's just github, but for me the psvm declaration and the entire @getmapping block looks much more indented than the method body

Reply:
oh i see what you mean. that's weird, i'll send a screenshot in the chat of what it looks like on my end